### ISSUE TITLE
.Top and .Skip are ignored for actions

### ISSUE THREAD

**Issue by nirinchev** [object/Simple.OData.Client#79]
The .Top and .Skip values are ignored when executing action. For example,

```
client.For<Foo>().Action("Bar").Skip(10).Top(10)
```

Will generate

```
Foo/Bar
``` 

instead of 

```
Foo/Bar?$skip=10&$top=10
```

**Comment by object** [comment]
Thank you for the report. Let me investigate this issue and get back to you.

**Comment by object** [comment]
Yes, I reproduced the problem. The fix is coming soon.

**Comment by object** [comment]
Fixed in version 4.2.1.
